adafruit_irremote.py

```diff
@@ -211,12 +216,12 @@ class NonblockingGenericDecode:
     ...         ...
     """
 
-    def __init__(self, pulses, max_pulse=10_000):
+    def __init__(self, pulses, max_pulse=10_000) -> None:
```

Comment:
`pulses`, and `max_pulse` here can have types as well. The number one on max_pulse I think is still an int even though it has the underscore in it. Not positive though.